File: leetcode/easy/2490/2490.cpp
#include<iostream>
#include<vector>
#include<set>
#include<string>
#include<queue>
#include<cstring>
using namespace std;
class Solution {
public:
    bool isCircularSentence(string sentence) 
    {
            char base = sentence[0];
            for(int i=1;i<sentence.size();i++)
            {
                if(sentence[i]==" ") 
                {
                    if(sentence[i-1] != sentence[i+1]) return 0;
                }
            }
            if(sentence[0]!=sentence[sentence.size()-1]) return 0;
            return 1;
    }
};

File: leetcode/easy/2490/2490.py
from typing import List

class Solution:
    def isCircularSentence(self, sentence: str) -> bool:
        strs : List[str] = sentence.split()
        lenth : int = len(strs)
        for i in range(lenth-1):
            if strs[i][-1] != strs[i+1][0]:
                return False
        if strs[-1][-1] != strs[0][0]:
            return False
        return True